library/src/main/java/com/google/android/exoplayer2/extractor/mp4/FragmentedMp4Extractor.java

```diff
@@ -1192,7 +1192,7 @@ private static DrmInitData getDrmInitDataFromAtoms(List<Atom.LeafAtom> leafChild
         if (uuid == null) {
           Log.w(TAG, "Skipped pssh atom (failed to extract uuid)");
         } else {
-          schemeDatas.add(new SchemeData(uuid, MimeTypes.VIDEO_MP4, psshData));
+          schemeDatas.add(new SchemeData(uuid, MimeTypes.CENC, psshData));
```

Comment:
You should also add this for the case where the pssh is parsed from the manifest. That is in `DashManifestParser#parseContentProtection`.
Otherwise if you follow @ojw28 tip about doing this in the DrmSessionManager, that would cover both cases.